### DISCUSSION TITLE
500: failed to create user account

### DISCUSSION THREAD

**Discussion by dpcode1** [AntlerVC/firetable#905] [Support / Q&A] [answered]
I am getting this error after successfully creating a project and applying all the firebase rules.  Please help!!!!
<img width="1016" alt="Screenshot 2022-10-30 at 12 27 07 PM" src="https://user-images.githubusercontent.com/116999539/198866504-d297165d-43fc-4a85-8467-2e13186f9c73.png">

**Comment by BeeBombshell** [comment, marked as answer]
Hello, @dpcode1. It's a little challenging to see how this isn't working out for you. It'd be great if you can set up a call with @shamsmosowi, so that he can help you figure this out. Here's a link to set up a support call: https://cal.com/shams/rowy
